### PR TITLE
Add page titles

### PR DESCRIPTION
**Pull request by johanbissemattsson**
### Tickets involved
[LWS-122](https://jira.kb.se/browse/LWS-122)

### Solves

Adds titles on pages; which are shown in the browsers title bar or a page's tab.
A separator (` | `) and suffix (`Libris`) is added when using the `getPageTitle` helper method.

### Summary of changes

- Add page titles

### CODE REVIEW

**Comment by johanbissemattsson** [comment]
Should we add a new lens for page titles? Currenty only the title of the work is shown but wouldn't the title + contributors be preferable (at least for books?

**Comment by olovy** [comment]
> Should we add a new lens for page titles? Currenty only the title of the work is shown but wouldn't the title + contributors be preferable (at least for books)?

This could maybe be a future improvement, yes

**Review by olovy — APPROVED** [review]
LGTM!

Limit the length (of the first part) of the title?

This old document [1] recommends 64 characters. Modern browsers have larger limits, e.g. Chrome has 1024.

[1] https://www.w3.org/Provider/Style/TITLE.html

**Review by olovy — APPROVED** [review]

File: lxl-web/src/lib/utils/getPageTitle.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Adds title separator and suffix to title string
+ */
+function getPageTitle(title?: string) {
+	if (title) {
+		return `${title} | Libris`;
+	}
+	return 'Libris';
```
[the remark this answers is not in the document]
Comment (johanbissemattsson) [reply lxl-web/src/lib/utils/getPageTitle.ts]: Added a comment about it in the latest commit

**Comment by johanbissemattsson** [comment]
> LGTM!
> 
> Result is a bit funny when using filters while searching.
> 
> Limit the length (of the first part) of the title? This old document [1] recommends 64 characters. Modern browsers have larger limits, e.g. Chrome has 1024. [1] https://www.w3.org/Provider/Style/TITLE.html

It now truncates the title if it is longer than the max length (currently set to 64).

**Pull request merged**